content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -744,10 +732,8 @@ it will become fully deprecated in a future Kubernetes release.
 
 {{< note >}}
 For most volume types, you do not need to set this field. It is automatically
-populated for [AWS EBS](/docs/concepts/storage/volumes/#awselasticblockstore),
-[GCE PD](/docs/concepts/storage/volumes/#gcepersistentdisk) and
-[Azure Disk](/docs/concepts/storage/volumes/#azuredisk) volume block types. You
-need to explicitly set this for [local](/docs/concepts/storage/volumes/#local) volumes.
+populated for [GCE PD](/docs/concepts/storage/volumes/#gcepersistentdisk) volume block types.
```

Comment:
Remove "GCE PD"

Reply:
According to https://github.com/kubernetes/website/pull/42302#discussion_r1282723644, GCE PD is still in Kubernetes 1.27